discretize/base/base_mesh.py

```diff
@@ -647,6 +653,89 @@ def _parse_location_type(self, location_type):
         "project_face_vector", "projectFaceVector", removal_version="1.0.0"
     )
 
+    @property
+    def axis_u(self):
+        """
+        .. deprecated:: 0.7.0
+          `axis_u` will be removed in discretize 1.0.0, it is replaced by
+          `mesh.orientation` for better mesh orientation validation.
+
+        See Also
+        --------
+        orientation
+        """
+        warnings.warn(
```

Comment:
It might be worth checking in on pyvista development here? If keeping these properties around simplifies interoperability, then it might be worth keeping them around?